docs/scanner-pools.md

```diff
@@ -0,0 +1,182 @@
+
+![Network Participants](NetworkParticipants.png)
+
+- Navigate to the node pool page by clicking on the address
+
+![Delegated Staking](DelegatedStaking.png)
+
+- Click on the “Delegate” button. There are 2 transactions required for delegation
+    - Approve the amount of FORT to delegate
+    - Delegate the approved FORT
+
+- Enter the amount of FORT you want to approve for delegation. Click “Approve tokens” and approve the transaction. Wait for the transaction to confirm.
+
+![Delegated Staking Approve](DelegatedStakingApprove.png)
+
+- Enter the amount of approved FORT you want to delegate and click “Delegate” and approve the transaction. Wait for the transaction to confirm
+
+![Delegated Staking FORT](DSFORT.png)
+
+- Navigate to “My Stakes” or “Overview” on your profile and you can see your staked FORT
+
+## Creating new nodes
+
+Each node needs to be registered to a specific pool in order to be operational. While one can register many scan nodes on a pool, the pool needs to be populated with more stake first in order to avoid registration issues. This is for preventing pool shutdown conditions that occur when the average per-scanner stake is below minimum.
+
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on the node pool you would like to add a node
+![My Node Pools List](MyNodePoolsList.png)
+
+- Make sure you have at least 2,500 FORT staked for the new node. Click “Add Scan Node”. Use the Forta node CLI to generate the scanner authorization token (described [here](scanner-quickstart#register-scan-node)) needed to claim a node into a pool.
+
+![Add Scan Node](AddScanNode.png)
+
+## Disabling existing nodes
+
+If a node is no longer used, it should be disabled to free up allocated stake for newer nodes.
+
+- Navigate to the node pool that has the node you want to disable
+
+- Select the disable action on the node you want to disable
+
+![Disable Scan Node](DisableScanNode.png)
+
+## Adjusting stake allocation
+
+While it's always possible to add more stake to a pool, it is allocated proportionally to the amount of scan nodes in the pool. Total allocated stake can only be `node_count × max_stake_per_scan_node`.
+
+For example, if there are two scan nodes in a pool, a total of 35k FORT deposited and only 15k FORT worth of max stake allocation is allowed per scan node, then the pool can allocate a maximum of `15k × 2 = 30k` FORT in total. This is the amount that is effective in calculating the rewards and the rest of `35k - 30k = 5k` remains unallocated until more scan nodes are registered to the pool.
+
+It is up to the pool owner to allocate more from owner stake or more from the delegators so the share of the rewards can be adjusted accordingly.
+
+For each node in a pool:
+
+- The operator needs to allocate at least 2500 FORT.
+- The total capacity is 15000 FORT.
```

Comment:
does this mean maximum stake? maybe we should call it that instead of "capacity"

Reply:
No, there is no upper bound on the stake anymore, this refers to the allocation capacity. But I am rewriting this part because I realized it even confuses myself while reading. Each of these numbers are about stake allocation capacity per node.